src/Payum/Stripe/StripeCheckoutGatewayFactory.php

```diff
@@ -16,7 +16,7 @@ class StripeCheckoutGatewayFactory extends GatewayFactory
      */
     protected function populateConfig(ArrayObject $config)
     {
-        if (!class_exists('Stripe')) {
```

Comment:
Installed dependencies on master, then switched to my branch and ran phpunit. This seems about right: 
```
/code/Payum/Payum/src/Payum/Stripe on bump-stripe λ phpunit
PHPUnit 4.8.21 by Sebastian Bergmann and contributors.

...............................................................  63 / 128 ( 49%)
.............................................EEEEEEFE....EEEEEE 126 / 128 ( 98%)
FE

Time: 163 ms, Memory: 11.00Mb

There were 14 errors:

1) Payum\Stripe\Tests\StripeCheckoutGatewayFactoryTest::shouldAllowCreateGateway
LogicException: You must install "stripe/stripe-php":"2.0 - 3.6" library.

/code/Payum/Payum/src/Payum/Stripe/StripeCheckoutGatewayFactory.php:20
/code/Payum/Payum/src/Payum/Stripe/vendor/payum/core/Payum/Core/GatewayFactory.php:45
/code/Payum/Payum/src/Payum/Stripe/vendor/payum/core/Payum/Core/GatewayFactory.php:33
/code/Payum/Payum/src/Payum/Stripe/Tests/StripeCheckoutGatewayFactoryTest.php:55
/usr/share/php/PHPUnit/TextUI/Command.php:147
/usr/share/php/PHPUnit/TextUI/Command.php:99
...
```